dev/breeze/tests/test_selective_checks.py

```diff
@@ -204,7 +204,7 @@ def assert_outputs_are_printed(expected_outputs: dict[str, str], stderr: str):
                     "run-kubernetes-tests": "true",
                     "upgrade-to-newer-dependencies": "false",
                     "parallel-test-types-list-as-string": "Always "
-                    "Providers[airbyte,apache.livy,dbt.cloud,dingding,discord,http]",
+                    "Providers[airbyte,apache.livy,dbt.cloud,dingding,discord,http,microsoft.azure]",
```

Comment:
What was the reason to add this?

Reply:
Because the CI breeze checks were failing with the below error

``AssertionError: Expected affected-providers-list-as-string=airbyte apache.livy dbt.cloud dingding discord http not found in affected-providers-list-as-string=airbyte apache.livy dbt.cloud dingding discord http microsoft.azure``

CI run:-
https://github.com/apache/airflow/actions/runs/4932512663/jobs/8815630158?pr=31134

Conversation thread [here](https://apache-airflow.slack.com/archives/CQ9QHSFQX/p1683686186361259)